core/misc/announce.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Adds an HTML element and method to trigger audio UAs to read system messages.
+ *
+ * Use Backdrop.announce() to indicate to screen reader users that an element on
+ * the page has changed state. For instance, if clicking a link loads 10 more
+ * items into a list, one might announce the change like this.
+ * $('#search-list')
+ *   .on('itemInsert', function (event, data) {
+ *     // Insert the new items.
+ *     $(data.container.el).append(data.items.el);
+ *     // Announce the change to the page contents.
+ *     Backdrop.announce(Backdrop.t('@count items added to @container',
+ *       {'@count': data.items.length, '@container': data.container.title}
+ *     ));
+ *   });
+ */
+(function (Backdrop, debounce) {
+  var liveElement;
+  var announcements = [];
+  Backdrop.behaviors.backdropAnnounce = {
+    attach: function attach(context) {
+      if (!liveElement) {
+        liveElement = document.createElement('div');
+        liveElement.id = 'backdrop-live-announce';
+        liveElement.className = 'visually-hidden';
```

Comment:
"visually-hidden" means that it's visible in Backdrop - which actually uses the class "element-invisible" for that.

Reply:
Thanks